app/views/works/_form_hidden_fields.html.erb

```diff
@@ -4,9 +4,9 @@
   initially and how many are added on the fly. This helps to process the data when the form
   is submitted.
   -->
-  <input type="text" id="existing_title_count" name="existing_title_count" value="<%= @work.datacite_resource.titles.count %>" class="hidden" />
+  <input type="text" id="existing_title_count" name="existing_title_count" value="<%= @work&.datacite_resource&.titles&.count || 0 %>" class="hidden" />
```

Comment:
The extra `&.` validations are needed because this view is now rendered from another view that does not declare an `@work`.